doc/content/gateways/lora-basics-station/lns.md

```diff
@@ -16,9 +16,9 @@ This page contains information about connecting your gateway to {{% tts %}} usin
 
 Follow instructions for [Adding Gateways in the Console]({{< ref "/getting-started/console/create-gateway" >}}) or [Adding Gateways Using the Command-line interface]({{< ref "/getting-started/cli/create-gateway" >}}).
 
-## Create an API Key (Optional)
+## Create an API Key
```

Comment:
I added the `new-in-version` tag because it is no longer optional. We could leave the shortcode and clarify in the text - "Since version 3.9, LNS requires an API Key"

It's good practice to highlight changes like this